client/src/tpu_client.rs

```diff
@@ -187,7 +187,9 @@ impl LeaderTpuCache {
     }
 
     fn fetch_slot_leaders(rpc_client: &RpcClient, start_slot: Slot) -> Result<Vec<Pubkey>> {
-        Ok(rpc_client.get_slot_leaders(start_slot, 2 * MAX_FANOUT_SLOTS)?)
+        let epoch_info = rpc_client.get_epoch_info()?;
```

Comment:
Let's cache `slots_in_epoch` please to reduce RPC server load, we don't need to be continually asking RPC for this value

Reply:
ok, now it should just call once per epoch.

Reply:
Thanks, that's much better than what I was thinking of :)